src/results.jl

```diff
@@ -82,7 +82,8 @@ function show_result(io::IO, rsp::TransactionResponse)
 
     for index in eachindex(rsp.metadata)
         println(io, rsp.metadata[index]["relationId"])
-        tuples = zip(rsp.results[index][2]...)
+        data = rsp.results[index][2]
+        tuples = isempty(data) ? [()] : zip(data...)
```

Comment:
Oops, this was an unrelated fix that snuck in here; i didn't mean to commit it. Lemme back that out and do it separately